src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is working right now!");
```

Comment:
The message 'Excavator is working right now!' could be more informative by specifying the type of machine in a more descriptive way, for example, 'Excavator is digging' to clearly reflect the specific work the Excavator is doing.